### ISSUE TITLE
Groups (via ActivityPub groups)

### ISSUE THREAD

**Issue by ArtificialOwl** [nextcloud/social#1007]
Managing ActivityPub's Group entity

**Comment by poVoq** [comment]
As Pleroma is also currently implementing this, it would probably be a good idea to cross check with them to avoid incompatibilities.

**Comment by jancborchardt** [comment]
@poVoq do you have more info on that, or are you working on it or know people who are? :)

**Comment by jancborchardt** [comment]
_[poVoq](/poVoq) wrote in #427:_ 

> **Is your feature request related to a problem? Please describe.**
> Would be nice to allow people to write to a group on Nextcloud social and have automatically a group channel that can be written to by using the !bang tag like in Friendica, Hubzilla or GNUSocial.
> 
> **Describe the solution you'd like**
> Compatible group / forum channel implementation similar to Friendica or Hubzilla on ActivityPub that you can follow via `*groupname*@yournextclouddomain.xyz` and write to via `!*groupname*@nextclouddomain.xyz`
> Groups should be of course created from the Nextcloud internal groups and/or maybe circles.
> 
> Thanks for considering.

**Comment by poVoq** [comment]
No, I am just going by what is mentioned on the official Pleroma blog about soon coming group support.

As of what I wrote in my previous message that was reposted before: it seems this isn't the preferred method anymore in ActivityPub. Going by explanations on the Hubzilla development group-channel the new method of doing groups in AP is to send private messages to the groups channel, which then reposts them on behalf of the user or so.

**Comment by jancborchardt** [comment]
@poVoq thanks for the info! :) Could you also share the relevant links, like the one from the Pleroma blog and if there’s any way to link to the relevant Hubzilla dev group channel messages?

**Comment by poVoq** [comment]
The issue as for Pleroma seems to be tracked here: https://git.pleroma.social/pleroma/pleroma/-/issues/656

**Comment by poVoq** [comment]
Can't find the Hubzilla thread anymore, but this document has some details:
https://framagit.org/zot/zap/-/blob/release/FEDERATION.md

**Comment by SchBenedikt** [comment]
PR #2291 reviewed the ActivityPub Groups request. Existing Nextcloud team actors do not implement AP Group semantics and cannot be represented as a complete solution; adding interoperable group collections and delivery is not part of this PR. No fix is claimed: https://github.com/nextcloud/social/pull/2291

**Comment by karlitschek** [comment]
fixed now
